build-db/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Use postgres 9.5
+FROM postgres:9.5
```

Comment:
I was referring to here. I believe we actually use some geometric functionalities from 10+.

Reply:
Updated image to 10.14.